README.md

```diff
@@ -74,8 +74,7 @@ Clone this repository to your desired folder:
 
 ### Usage
 
```

Comment:
- [ **OPTIONAL** ] Kindly ensure that qll main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. Right now, I noticed that you miss the **Future Features** section, including 1-3 potential features to be added later.